libraries/c_sdk/aws/shadow/include/types/aws_iot_shadow_types.h

```diff
@@ -471,7 +471,7 @@ typedef struct AwsIotShadowDocumentInfo
              * This only needs to be set if #AWS_IOT_SHADOW_FLAG_WAITABLE is passed to
              * @ref shadow_function_get.
              */
-            void *( *mallocDocument )( size_t );
+            void * ( *mallocDocument )( size_t );
         } get; /**< @brief Valid members for @ref shadow_function_get. */
```

Comment:
Same for this and a couple of other files, is this an uncrustify problem? This does bloat this PR with lots of files?

Reply:
Yes our uncrustify config file doesn't like the * to be attached to anything, it depends on the version i was using as well 1.7 instead of 1.6